src/app/WriteHandler.cpp

```diff
@@ -321,6 +321,13 @@ CHIP_ERROR WriteHandler::ProcessAttributeDataIBs(TLV::TLVReader & aAttributeData
         err = element.GetData(&dataReader);
         SuccessOrExit(err);
 
+        if (!dataAttributePath.IsValid())
+        {
+            err = AddStatus(dataAttributePath, StatusIB(Status::InvalidAction));
```

Comment:
InvalidAction is not a sensical path-specific status....

If we want to treat this as "invalid action, abort all further processing", then we need to abort the further processing.  If we want to treat this as a path-specific error, what is the spec basis for a special check here?  I don't see one.

Reply:
I see, I would suggest not to abort the processing, but let's discuess in the spec thread.